src/components/info-tile/info-tile.tsx

```diff
@@ -0,0 +1,265 @@
+import { Component, Prop, h } from '@stencil/core';
+
+/**
+ * This component can be used on places such as a start page or a dashboard.
+ * It offers features for visualizing aggregated data along with supplementary
+ * information.
+ *
+ * Clicking on the component should navigate the user to
+ * a new screen or web page, to which you need to provide a URL,
+ * using the `href` property.
+ *
+ * By default, this navigation will happen within the same browser tab.
+ * However, it is possible to override that behavior,
+ * using the `target` property.
+ *
+ * @exampleComponent limel-example-info-tile
+ * @exampleComponent limel-example-info-tile-notification
+ * @exampleComponent limel-example-info-tile-progress
+ * @exampleComponent limel-example-info-tile-loading
+ * @exampleComponent limel-example-info-tile-styling
+ */
+
+@Component({
+    tag: 'limel-info-tile',
+    shadow: true,
+    styleUrl: 'info-tile.scss',
+})
+export class InfoTile {
+    /**
+     * Name of icon for the info-tile.
+     */
+    @Prop()
+    public icon?: string;
+
+    /**
+     * The text to show below the info-tile. Long label will be truncated.
+     */
+    @Prop({ reflect: true })
+    public label?: string = null;
+
+    /**
+     * A string of text that is visually placed before the value.
+     */
+    @Prop({ reflect: true })
+    public prefix?: string;
+
+    /**
+     * A piece of text or number that is the main piece of information
+     * which the component is intended to visualize.
+     */
+    @Prop({ reflect: true })
+    public value: number | string;
+
+    /**
+     * A string of text that is visually placed after the value.
+     */
+    @Prop({ reflect: true })
+    public suffix?: string;
+
+    /**
+     * The `title` tag of the hyperlink, which can be used to
+     * provide additional information about the link.
+     * It improves accessibility both for sighted users
+     * and users with assistive technologies.
+     */
+    @Prop({ reflect: true })
+    public linkTitle?: string = null;
+
+    /**
+     * Set to `true` if info-tile is disabled.
+     */
+    @Prop({ reflect: true })
+    public disabled: boolean = false;
+
+    /**
+     * The url that the info-tile leads to.
+     */
+    @Prop({ reflect: true })
+    public href?: string = null;
+
+    /**
+     * Set to `true` if info-tile is clickable.
+     */
+    @Prop({ reflect: true })
+    public clickable: boolean = false;
```

Comment:
Do we need this? If an `href` is provided, I would assume it will be clickable and not clickable if omitted

Reply:
It is possible to use the component for visualization purposes only. So there would be no need for the link `<a` functionality in that case, and or clickability. 

So, when there is a link (`href`) this adds an elevated effect using a shadow, as well as `cursor: pointer` by applying a class. So the component looks clickable. 

But the component can also be used without a link (`href`), in another webcomponent for instance, which only sets an `onClick` actions from outside. We can't know this from inside the component. That's why we made this a prop.

Reply:
Added a fixup with improved docs

Reply:
Ok! If it is for visual purposes only, can we make it with a CSS prop instead? Or maybe a class name?

Reply:
But should we even have the link functionality built into this component then? Maybe that is just making it too smart? The components in Lime Elements are supposed to be as dumb as possible to make them reusable, so maybe that functionality should be handled by the consumer instead.

Reply:
Yes, using a class name would also work the same in this case I think. But my experience helping the appers shows that developers use props more properly than class names. So for instance, they'd more easily set a prop to true programmatically, rather than applying a class name programatically.

Additionally, these styling CSS classes would need an unnecessary example too. They won't be reflected in our composite examples neither.

I feel we have done more example like this before in Elements. Right now too tired to remember them. Or to say for sure that a class name is better than a prop. Maybe we also need it for more things than mere styling.

Reply:
I understand Johan's argument, and using a CSS class or CSS variable ought to be a good bit more performant, but I also see the point in props being easier to get people to use. If I completely disregard any performance aspect, I think I actually prefer to have it as a prop.

However, going from Johan's initial question—"Do we need this?"—and the answer that this can be used only for visualisation: can you elaborate? I understand that it _could_ be used for just about anything, but the question is whether it _should_. Is there a legitimate use case where a component called _shortcut_ should be used to only visualise something?

Reply:
Crap, scratch my above comment… as you will have already figured out, I was confused 🤦‍♂️ 🤣

Reply:
⚡